identity/src/main/java/com/stripe/android/identity/injection/IdentityVerificationScope.kt

```diff
@@ -0,0 +1,10 @@
+package com.stripe.android.identity.injection
+
+import javax.inject.Scope
+
+/**
+ * Scope for an verification, used for [IdentityActivitySubcomponent].
+ */
+@Scope
+@Retention
+internal annotation class IdentityVerificationScope
```

Comment:
the scope ensures all dependencies in `IdentityActivitySubcomponent` are unique

Before the change there are multiple `IdentityAnalyticsRequestFactory` instances, making the `lateinit var` still not initialized when the network call returns.